spec/generators/graphql/install_generator_spec.rb

```diff
@@ -3,20 +3,19 @@
 require "generators/graphql/install_generator"
 
 class GraphQLGeneratorsInstallGeneratorTest < Rails::Generators::TestCase
-
   tests Graphql::Generators::InstallGenerator
   destination File.expand_path("../../../tmp/dummy", File.dirname(__FILE__))
 
   setup do
     prepare_destination
-    FileUtils.cd(File.expand_path("../../../tmp", File.dirname(__FILE__))) do
-      `rm -rf dummy`
+
+    FileUtils.cd(File.join(destination_root, '..')) do
       `rails new dummy --skip-active-record --skip-test-unit --skip-spring --skip-bundle`
     end
   end
```

Comment:
I thought you meant some spec helper that you can call when dealing with a spec that needs Rails power.